code/modules/admin/verbs/randomverbs.dm

```diff
@@ -1208,4 +1208,59 @@ var/list/datum/outfit/custom_outfits = list() //Admin created outfits
 				A.temperature = T20C
 	message_admins("[key_name(src)] filled the hullbreachs in [size] tiles.")
 	log_game("[key_name(src)] filled the hullbreachs in [size] tiles.")
-	*/
\ No newline at end of file
+	*/
+	
+/*
+
+	When you want tell something important to the aliens
+
+*/
+/client/proc/cmd_admin_create_xeno_report()
+	set category = "Special Verbs"
+	set name = "Create Alien Report"
+	if(!holder)
+		src << "Only administrators may use this command."
+		return
+	var/input = input(usr, "Please enter anything you want. Anything. Serious.", "What?", "") as message|null
```

Comment:
this is not safe and it'd allow an admin to gain access with hrefs, try to find a proc that doesn't allow it(should be stripped_input, not totally sure)